src/modules/75-cf/components/FlagActivation/ServicesList.module.scss

```
@@ -8,12 +8,18 @@
 .displayTags {
   background: var(--primary-2);
   margin: 4px 0;
```

You can use spacing var here:
```suggestion
  margin: var(--spacing-xsmall) 0;
```

---

good spot!